samples/sampler/stories/default/Icon.js

```diff
@@ -26,38 +26,45 @@ const SkinnedIconBase = kind({
 
 	render: ({skin, ...rest}) => {
 		let iconNames;
-		const size = select('size', ['smallest', 'small', 'large', 'huge'], Config, 'large');
+		const size = rest['size'];
 
 		switch (skin) {
 			case 'silicon': {
-				iconNames = iconListSilicon; break;
+				iconNames = iconListSilicon;
+				break;
 			}
 			default: {
 				iconNames = iconList;
 			}
 		}
 
-		const iconType = select('icon type', ['glyph', 'url src', 'custom'], Config, 'glyph');
+		const iconType = rest['iconType'];
 		let children;
 
 		switch (iconType) {
-			case 'glyph': children = select('icon', ['', ...iconNames], Config, 'home'); break;
-			case 'url src': children = select('src', ['', docs, factory, logo], Config, logo); break;
-			default: children = text('custom icon', Config);
+			case 'glyph':
+				children = rest['icon'];
+				break;
+			case 'url src':
+				children = rest['src'];
+				break;
+			default:
+				children = rest['customIcon'];
 		}
 
```

Comment:
we need here now a "delete rest.iconTrype"  to avoid passing it to the child component